### DISCUSSION TITLE
lerna - possible to use a main as default branch name instead of master?

### DISCUSSION THREAD

**Discussion by dman777** [npm/feedback#269] [CLI] [answered]
For lerna, is it possible to use the default name `main` instead of `master` with lerna init? I tried to change the branch name manually but I get 

```
% git branch -m main
error: refname refs/heads/master not found
fatal: Branch rename failed
```

**Comment by MylesBorins** [comment, marked as answer]
hey @dman777 the npm team does not manage or maintain lerna. Your best bet will be opening an issue over at https://github.com/lerna/lerna
